README.md

```diff
@@ -107,6 +107,57 @@ DaskTaskRunner(
 )
 ```
 
+### Distributing Dask collections across workers
+
+If your task contains a Dask collection, such as a `dask.DataFrame` or `dask.Bag`, to distribute the work across workers and achieve parallel computations, use the `get_dask_client` context manager.
+
+Within task run contexts:
+```python
+import dask
+from prefect import flow, task
+from prefect_dask import DaskTaskRunner, get_dask_client
+
+@task
+def compute_task():
+    with get_dask_client(timeout="120s") as client:
+        df = dask.datasets.timeseries("2000", "2001", partition_freq="4w")
+        summary_df = client.compute(df.describe()).result()
+    return summary_df
+
+@flow(task_runner=DaskTaskRunner())
+def dask_flow():
+    prefect_future = compute_task.submit()
+    return prefect_future.result()
+
+dask_flow()
+```
+
+Within flow run contexts; `timeout` must be set in `DaskTaskRunner`:
+```python
+import dask
+from prefect import flow
+from prefect_dask import DaskTaskRunner, get_dask_client
+
+@flow(task_runner=DaskTaskRunner(client_kwargs=dict(timeout="120s")))
+def dask_flow():
+    with get_dask_client() as client:
+        df = dask.datasets.timeseries("2000", "2001", partition_freq="4w")
+        summary_df = client.compute(df.describe()).result()
+    return summary_df
```

Comment:
To make sure I'm understanding this correctly, is there any functional difference between this example and:

```python
import dask
from dask.distributed import Client, LocalCluster
from prefect import flow

@flow()
def dask_flow():
    df = dask.datasets.timeseries("2000", "2001", partition_freq="4w")
    return df.describe().compute()

with LocalCluster() as cluster:
    client = Client(cluster)
    dask_flow()
```

It seems not using `get_dask_client` is simpler and requires less advanced knowledge of Dask in this case. It is also entirely possible there are nuances I'm not understanding on the Prefect side!

Reply:
I think by using `DaskTaskRunner`, users don't have to instantiate LocalCluster and Client themselves

Reply:
Yep! When running a flow via a deployment we'd bypass code that does setup of a Dask cluster like that so we need to provide a way for the cluster to be allocated by the flow itself.

Reply:
Ah great, that makes sense. Thank you both for the explanation!